crates/eip5792/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "alloy-eip5792"
+description = "Types for the 'wallet' Ethereum JSON-RPC namespace"
+
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
```

Comment:
this we can't do because now we need to vendor them individually so all of them must be assigned the current alloy version